mantis-common/src/main/java/io/mantisrx/common/compression/CompressionUtils.java

```diff
@@ -217,10 +217,19 @@ private static int getTotalByteSize(List<List<byte[]>> nestedEvents, byte[] deli
             for (int i = 0; i < line.length(); i++) {
```

Comment:
+1 to this comment. perhaps there is a way to achieve this using regex or something.